src/translations/translations_EN.json

```diff
@@ -47,10 +47,9 @@
     "license_information": "License information",
     "modify_and_build": "Modify and build upon",
     "modify_and_build_options": {
-      "dont_care": "And, they may choose their own license",
       "ND": "No, others can only use the material in it's original form",
       "primary_yes": "Yes, others can modify and build upon your material",
-      "share_alike": "But, they have to use the same license as this resource (be aware that this might cause complications for usage of your resource)"
+      "share_alike": "Yes, but they have to use the same license as this resource (be aware that this might cause complications for usage of your resource)"
```

Comment:
Tenker egentlig at det som står i parantes burde trekkes ut og heller plassert i en litt mindre typography under "yes, but they have to use the same license as this resource"

Reply:
Egentlig enig, men jeg ser ikke hvordan vi skal få det til uten å lage egen komponent for radio-knappene. FormControlLabel støtter ikke stiling inne label-attributtet

Reply:
Man får lov hvis man er sta nok:
![image](https://user-images.githubusercontent.com/44363838/107212896-ebe68300-6a07-11eb-873e-f14e54896f10.png)

                label={
                  <>
                    <Typography variant="h5">
                      {t(`license.modify_and_build_options.${DefaultModifyAndBuildOptions.ND}`)}
                    </Typography>
                    <Typography variant="h1">
                      {t(`license.modify_and_build_options.${DefaultModifyAndBuildOptions.ND}`)}
                    </Typography>
                  </>
                }

Reply:
du altså :-D